tiac/templates/tiac/_investigation_tiac_informations_synthese.html

```diff
@@ -19,49 +19,68 @@
-                    </div>
-                {% endif%}
+                <div class="fr-col-4 bold">Etablissement(s)</div>
+                <div class="fr-col-8">
+                    {% if raisons_sociales|length == 0 %}
+                        <span class="empty-value">Vide</span>
+                    {% elif raisons_sociales|length == 1 %}
+                        {{ raisons_sociales.0 }}
+                    {% else %}
+                        {{ raisons_sociales.0 }}
+                        <a href="#" class="fr-link no-underline" aria-describedby="message-tooltip-raisons">
+                            <p class="fr-tag">et
+                                {% if raisons_sociales|length|add:"-1" == 1 %}
+                                    1 autre
+                                {% else %}
+                                    {{ raisons_sociales|length|add:"-1" }} autres
+                                {% endif %}
+                            </p>
+                        </a>
+                        <span class="fr-tooltip fr-placement" id="message-tooltip-raisons" role="tooltip" aria-hidden="true">Autres établissements : {{ raisons_sociales|slice:"1:"|join:', ' }}</span>
+                    {% endif%}
+                </div>
             </div>
             <div class="fr-grid-row fr-grid-row--gutters">
-                <div class="fr-col-7 bold">Commune(s)</div>
-                <div class="fr-col-5">
+                <div class="fr-col-4 bold">Commune(s)</div>
+                <div class="fr-col-8">
                     {% if communes|length == 0 %}
-                        <div class="fr-col-5"><span class="empty-value">Vide</span></div>
+                        <span class="empty-value">Vide</span>
                     {% elif communes|length == 1 %}
-                        <div class="fr-col-5">{{ communes.0 }}</div>
+                        {{ communes.0 }}
                     {% else %}
-                        <div class="fr-col-5">{{ communes.0 }}
-                            <button class="fr-btn--tooltip fr-btn" type="button" id="button-communes" aria-describedby="tooltip-communes">
-                                Information contextuelle
-                            </button>
-                            <span class="fr-tooltip fr-placement" id="tooltip-communes" role="tooltip" aria-hidden="true">Autre communes : {{ communes|slice:"1:"|join:', ' }}</span>
-                        </div>
+                        {{ communes.0 }}
+                        <a href="#" class="fr-link no-underline" aria-describedby="message-tooltip-communes">
+                            <p class="fr-tag">et
+                                {% if communes|length|add:"-1" == 1 %}
+                                    1 autre
+                                {% else %}
+                                    {{ communes|length|add:"-1" }} autres
+                                {% endif %}
+                            </p>
```

Comment:
Pareil, qu'est-ce qeu tu dis de :
```suggestion
                            {% with rest_length = communes|length|add:"-1" %}
                                <p class="fr-tag">et {{ rest_length }} autre{{ rest_length|pluralize }}</p>
                            {% endwith %}
```